src/Controls/src/Core/Handlers/Items/CollectionViewHandler.cs

```diff
@@ -29,6 +29,8 @@ public CollectionViewHandler(PropertyMapper mapper = null) : base(mapper ?? Mapp
 			[Controls.ItemsView.IsVisibleProperty.PropertyName] = MapIsVisible,
 			[Controls.ItemsView.ItemsUpdatingScrollModeProperty.PropertyName] = MapItemsUpdatingScrollMode,
 			[StructuredItemsView.HeaderTemplateProperty.PropertyName] = MapHeaderTemplate,
+			[StructuredItemsView.HeaderProperty.PropertyName] = MapHeaderTemplate,
+			[StructuredItemsView.FooterProperty.PropertyName] = MapFooterTemplate,
```

Comment:
Shouldn't this be added to StructuredItemsViewHandler also? 
Or in theory it should be added there and can be removed here.